src/Microsoft.ML/ILearningPipelineItem.cs

```diff
@@ -14,6 +14,7 @@ namespace Microsoft.ML
     public interface ILearningPipelineItem
     {
         ILearningPipelineStep ApplyStep(ILearningPipelineStep previousStep, Experiment experiment);
+        Var<IDataView> GetInputData();
```

Comment:
>GetInputData [](start = 23, length = 12)

Please add xml comment. Not 100% clear what this fucntion supposed to do:  input to the item? When will this be useful?

Reply:
>When will this be useful
Is still remain open question

---
In reply to: [190401291](https://github.com/dotnet/machinelearning/pull/212#discussion_r190401291) [](ancestors = 190401291)